core/database/foxx/api/note_router.js

```diff
@@ -87,9 +99,32 @@ router
                         results: [note.new],
                         updates: Object.values(updates),
                     });
+                    ({ _key, _rev, ...result } = doc);
+                    logger.logRequestSuccess({
+                        client: client?._id,
+                        correlationId: req.headers["x-correlation-id"],
+                        httpVerb: "POST",
+                        routePath: basePath + "/create",
+                        status: "Success",
+                        description: "Create an annotation on an object",
+                        extra: result,
+                    });
                 },
             });
         } catch (e) {
+            if (doc) {
+                ({ _key, _rev, ...result } = doc);
+            }
+            logger.logRequestFailure({
+                client: client?._id,
+                correlationId: req.headers["x-correlation-id"],
+                httpVerb: "POST",
+                routePath: basePath + "/create",
+                status: "Failure",
+                description: "Create an annotation on an object",
+                extra: null,
```

Comment:
```suggestion
                extra: result,
```

The whole point was to output it right?